lib/MusicBrainz/Server/Email.pm

```diff
@@ -524,20 +524,20 @@ $message
 EOF
     }
 
-    my $admin_addresses = join ', ', map { _user_address($_) } @{ $opts{admins} };
+    my $admin_address = 'MusicBrainz Account Admins <musicbrainz-account-admin@metabrainz.org>'
```

Comment:
Should this maybe be moved to a constant or `DBDefs`?

Reply:
Yeah, I would move it to lib/MusicBrainz/Server/Constants.pm together with the other EMAIL_* addresses.

Reply:
Should be fixed as of https://github.com/metabrainz/musicbrainz-server/commit/0eb2f810dfcf71840c6cbcfa898d49054e42335a